src/main/java/core/basesyntax/service/csvfileservice/ReaderService.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.service.csvfileservice;
+
+import core.basesyntax.model.FruitTransaction;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ReaderService {
```

Comment:
you should create a separate interface with such a name and implement it here, right now it's a violation of the second principle of SOLID

Reply:
interfaces created